covalent/_data_store/models.py

```diff
@@ -179,7 +179,7 @@ class ElectronDependency(Base):
     arg_index = Column(Integer, nullable=True)
 
     # Name of the column which signifies soft deletion of the electron dependencies corresponding to a lattice
-    is_active = Column(Boolean, nullable=False)
+    is_active = Column(Boolean, nullable=False, default=True)
```

Comment:
@FyzHsn @scottwn wanted to check if these defaults for `is_active` are sensible

Reply:
Yeah, only when the user deletes a workflow in the UI can the field be changed to False